src/loadsave.jl

```diff
@@ -159,3 +159,51 @@ function loadtodict!(d::Dict, g::Union{JLDFile,Group}, prefix::String="")
     end
     return d
 end
+
+"""
+    save_object(filename, x)
+
+Write one object `x` from the current scope to a JLD2 file `filename`. The
+object is stored with a default name of "single_stored_object". If the default
+name needs to be overridden use [`@save`](@ref) instead. If the file exists, it
+overwrites it.
```

Comment:
```suggestion
Stores an object `x` in a new JLD2 file at `filename`. 
If a file exists at this path, it will be overwritten.

Since the JLD2 format requires that all objects have a name, the object will be stored as 
`"single_stored_object"`. 
If you want to store more than one object, use [`jldopen`](@ref), the FileIO API
or the [`@save`](@ref) macro.
```
I've rephrased this a bit to mention all the alternative ways of writing JLD2 files (of which, at least in my personal opinion, `@save` is probably the worst).

I've also added a rather clunky explanation for why the object has a name, although I'm not sure this is actually relevant in this docstring. Feel free to improve (or remove)!